doc/source/release/1.16.0-notes.rst

```diff
@@ -41,37 +41,69 @@ New features
 ``scipy.differentiate`` improvements
 ====================================
 
-
 ``scipy.interpolate`` improvements
 ==================================
+- Improved the error estimate of `scipy.integrate.tanhsinh`
 
+``scipy.interpolate`` improvements
+==================================
+- Added batch support to `scipy.interpolate.make_smoothing_spline`
```

Comment:
I think `interpolate` improvement section title here is duplicated from above, unless I'm just misreading this somehow. Can fuse easily enough.

Reply:
Ooops https://github.com/scipy/scipy/pull/22991/files#r2093784293